CHANGELOG.md

```diff
@@ -221,4 +222,5 @@ history, but any notable change from this point on shall be captured by this doc
 For archeological inquiries, check out the state of the repository at the time of the first
 changelog:
 
-[https://github.com/tigerbeetle/tigerbeetle/](https://github.com/tigerbeetle/tigerbeetle/tree/d2d6484188ecc57680e8bde446b5d09b6f2d83ca)
+[https://github.com/tigerbeetle/tigerbeetle/](
+https://github.com/tigerbeetle/tigerbeetle/tree/d2d6484188ecc57680e8bde446b5d09b6f2d83ca)
```

Comment:
Hard-wrapping links is annoying, but keeping the lint simple is probably more useful for the time being.
Maybe reconsider adding some exceptions if we ever need to include a URL that is >100 chars.